controllers/ingressController.go

```diff
@@ -57,10 +50,7 @@ func ShowAllowedIngress(c *gin.Context) {
 //WhiteListIP ...
 func WhiteListIP(c *gin.Context) {
 	var leases []models.Leases
-	errs := 0
-	var err error
-	var myIngress, data pkg.IngressList
-	updateStatusflag, updateStatus := false, true
+
 	User, _ := c.Get("User")
 	ns := c.Param("ns")
```

Comment:
instead of using ns to identify the driver. Can we introduce a new param called driver. We can add a to do that  as well.

Reply:
discussed. adding a todo and changes will be separate PR. (as quite a few places require this change)